themes/ctl-histologylab/static/scss/main.scss

```diff
@@ -1,2 +1,9 @@
+// The line below includes Bootstrap source scss files, which are built
+// with tooling in the Makefile. If this needs to be updated the directory
+// named 'bootstrap-4.0.0-beta.3' is equivalent to 'bootstrap-4.0.0-beta.3/scss'
+// (or whatever future version of Bootstrap that you're using) from the root
+// directory of the repo: https://github.com/twbs/bootstrap
+@import 'bootstrap-4.0.0-beta.3/bootstrap';
+
```

Comment:
This PR is hard to review because of the 80-something scss files from Bootstrap.  Essentially though, this is where everything is included.